### PR TITLE
Hp 66

### PR DESCRIPTION
**Pull request by Audiosutras**
https://caktus.atlassian.net/browse/HP-66

### CODE REVIEW

File: apps/hip/templates/tags/contact_info.html [hunk 1]
```diff
@@ -18,11 +18,11 @@ <h6 class="column is-11 is-size-6">Fax a report to <strong>{{ info.business_hour
     </div>
     <div class="columns is-vcentered is-mobile pt-2 px-4 contact-container-hip">
       <div class="column is-3 is-mobile dark-blue-bg-hip has-text-centered mr-1 is-full-height-hip">
-        <span class="is-block moon-icon-hip is-size-3 pt-4"></span>
+        <span class="is-block moon-icon-hip is-size-3 pt-5"></span>
         <h6 class=" column is-size-6">After Hours</h6>
       </div>
       <div class="column is-9 columns is-mobile light-grey-bg-hip ml-1 is-full-height-hip">
-        <span class="column is-1 is-block phone-icon-hip"></span>
+        <span class="column is-1 is-block pt-4 is-block phone-icon-hip"></span>
```
Comment (dchukhin) [inline apps/hip/templates/tags/contact_info.html:25]: It looks like `is-block` gets duplicated here now

**Review by dchukhin — COMMENTED** [review]
The icons look good now, except I think the phone-icon-hip on the after hours line still needs some adjustment. I'll post a screenshot of what it looks like for me
Screen Shot 2021-02-25 at 21 26 33

File: apps/hip/templates/tags/contact_info.html (hunk 1 above)
Comment (Audiosutras) [reply apps/hip/templates/tags/contact_info.html]: done

**Review by dchukhin — APPROVED** [review]
Matches wireframes! 🖼️

**Pull request merged**